Islandora-Devops/claw-playbook
# Use external roles

Resolves https://github.com/Islandora-CLAW/CLAW/issues/764

This moves all the internal roles out to the external ones.

This has introduced a bug [documented here](https://github.com/Islandora-CLAW/CLAW/issues/764#issuecomment-353077532) where you (or at least I) can't download all the external roles without a failure from galaxy (or Github...which is my guess).

### How To Test
1. Clone this branch to a new directory.
2. Run `vagrant up`. It will fail (or it always does for me on the 21st role)
3. Run `vagrant provision`

You should have a working CLAW instance.
